apps/src/music/blockly/setup.ts

```diff
@@ -42,6 +43,15 @@ export function setUpBlocklyForMusicLab() {
   const typedMusicBlocks = MUSIC_BLOCKS as {[key: string]: BlockConfig};
   for (const blockType of Object.keys(typedMusicBlocks)) {
     const blockConfig = typedMusicBlocks[blockType] as BlockConfig;
+
+    // Patch up the default sound if the library provides one.
+    if (
+      blockType === BlockTypes.PLAY_SOUND_AT_CURRENT_LOCATION_SIMPLE2 &&
```

Comment:
Is this something we can handle inside of `FieldSounds` (i.e. check if options.currentValue is undefined, and fallback to defaultSound if so)? I'd want to avoid deferring global blockly setup if possible

Reply:
Glad you asked.  I had tried something like this at first, but focused on the wrong place.  Figured it out this time, and it seems nice and clean.